README.md

```diff
@@ -378,6 +378,36 @@ loadedKMod : [<kernel module>, ...]
  Matching is done by performing logical _AND_ for each provided Element, i.e the Rule will match if all provided Elements (kernel modules) are loaded
  in the system.
 
+##### CpuId Rule
+###### Nomenclature
+```
+Element     :A CPUID flag
+```
+
+The Rule allows matching the in the system against a provided list of Elements.
+
+###### Format
+```yaml
+cpuId : [<CPUID flag string>, ...]
+```
+ Matching is done by performing logical _AND_ for each provided Element, i.e the Rule will match if all provided Elements (CPUID flag strings) are available
+ in the system.
+
+##### Kconfig Rule
+###### Nomenclature
+```
+Element     :A Kconfig option
+```
+
+The Rule allows matching the in the system against a provided list of Elements.
```

Comment:
> the in the system against a provided list of Elements.
the running kernel KConfig config options against a provided list of elements ?

Reply:
copy&paste issue. now fixed